bin/chat_simple_gui_client_log.log

```diff
@@ -0,0 +1,10 @@
+May 18, 2017 6:32:00 PM codeu.chat.SimpleGuiClientMain main
```

Comment:
Not related to this particular pull, but you might want to add all the .log files into your .gitignore

or even better, ignore everything under bin/ (which will have all the output files)